content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Server Software
+description: A detailed guide on which server software is right for you
+---
+
+import { Card, Cards } from "fumadocs-ui/components/card";
+
+# Introduction
+
+Everybody has different requirements when setting up their server. This guide will help you decide which server software is right for you.
+Each server software has its own uses and features, so it's important to choose the right one for your needs.
+
+<Callout type="info" title="Note">
+  Server Software(s) are commonly referred to as "Server Jar(s)". This is
+  because server softwares are usually distributed as a `.jar` file.
+</Callout>
+
+# Backend Servers
+
+Backend servers are the most common type of server software.
+There are an incomprehensible amount of backend servers available, however this section will cover the most recommended, stable and widely-used softwares.
+
+<Callout type="info" title="Note">
+  A "fork" is a copy of a software project that is modified in some way.\
+  They typically share the same codebase but have modified (or additional) features and optimizations.\
+  An example of "forks" include: Bukkit -> CraftBukkit -> Spigot -> Paper -> Pufferfish -> etc.
+</Callout>
+
+## Vanilla
+
+Vanilla is the official server software for Minecraft. As the name implies, it is the most basic server software available and has no modifications whatsoever.\
+It is ideal for those who wish to host a small server for a couple of friends or family, **without** altering any mechanics.\
+This server software is **not** recommended for larger servers.
+
+<Card href="https://jar.setup.md/download/vanilla/latest" title="Download">
+  Download a copy of the latest Vanilla server software via jar.setup.md.
+</Card>
+
```

Comment:
Should we not have at least a mention of CraftBukkit as progenitor of all derivative forks and originator of the API?

Likewise, should we not also mention Spigot? It is absolutely foolish to be using Spigot in 2024, but a lot of people still do, so for education's sake, it should be listed. Last thing we need is a post on the subreddit by someone saying "I read the wiki page for server software and thought I wanted to use Purpur, but I see a bunch of people talking about Spigot. Is that better? Should I use that instead?"

Reply:
Resolved in [bf3319b](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/bf3319b76bf41d657ef572b076edcff7516ae2b6)